collections/sentinel-2-l2a.yaml

```diff
@@ -166,6 +166,210 @@ CubeDimensions:
       - viewAzimuthMean
       - viewZenithMean
       - dataMask
-
-RegistryEntryAdded: "2018-04-17"
-RegistryEntryLastModified: "2021-08-18"
+sci:citation: Modified Copernicus Sentinel data [Year]/Sentinel Hub
+Summaries:
+  eo:bands:
+    - name: B01
+      common_name: coastal
+      center_wavelength: 0.4427
+      full_width_half_max: 0.021
+    - name: B02
+      common_name: blue
+      center_wavelength: 0.4924
+      full_width_half_max: 0.066
+    - name: B03
+      common_name: green
+      center_wavelength: 0.5598
+      full_width_half_max: 0.036
+    - name: B04
+      common_name: red
+      center_wavelength: 0.6646
+      full_width_half_max: 0.031
+    - name: B05
+      center_wavelength: 0.7041
+      full_width_half_max: 0.015
+    - name: B06
+      center_wavelength: 0.7405
+      full_width_half_max: 0.015
+    - name: B07
+      center_wavelength: 0.7828
+      full_width_half_max: 0.02
+    - name: B08
+      common_name: nir
+      center_wavelength: 0.8328
+      full_width_half_max: 0.106
+    - name: B8A
+      common_name: nir08
+      center_wavelength: 0.8647
+      full_width_half_max: 0.021
+    - name: B09
+      common_name: nir09
+      center_wavelength: 0.9451
+      full_width_half_max: 0.02
+    - name: B10
```

Comment:
I think S2L2A on SH doesn't have B10 bands.

Reply:
True, I just copied it from the file I created a while ago (mainly based on [our documentation](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l2a/)). It was already wrong in there but no one every noticed. Removed